src/LiveComponent/doc/index.rst

```diff
@@ -2236,37 +2236,92 @@ To validate only on "change", use the ``on(change)`` modifier:
 Deferring / Lazy Loading Components
 -----------------------------------
 
+When a page loads, all components are rendered immediately. If a component is
+heavy to render, you can defer its rendering after the page has loaded.
+
+Moreover, some components are not needed until the user scrolls to them (like
+components at the bottom of a long page like comments, related products, etc.).
+In this case, you can load them "lazily" when they are in the viewport.
+
+By loading a component asynchronously, the page will load faster and the user
+will be able to interact with it sooner, improving both the user experience
+and the overall performance of your application.
+
+.. note::
+
+    Behind the scenes, your component *is* created & mounted during the initial
+    page load, but it isn't rendered. So keep your heavy work to methods in
+    your component (e.g. ``getProducts()``) that are only called when rendering.
+
+Loading "defer"
+~~~~~~~~~~~~~~~
+
 .. versionadded:: 2.13.0
 
     The ability to defer loading a component was added in Live Components 2.13.
 
 If a component is heavy to render, you can defer rendering it until after
-the page has loaded. To do this, add the ``defer`` option:
+the page has loaded. To do this, add a ``loading="defer"`` attribute:
 
 .. code-block:: html+twig
 
-    {# With the HTML syntax #}
-    <twig:SomeHeavyComponent defer />
-
     {# With the component function #}
-    {{ component('SomeHeavyComponent', { defer: true }) }}
+    <twig:SomeHeavyComponent loading="defer" />
+
+.. code-block:: twig
+
+    {# With the HTML syntax #}
+    {{ component('SomeHeavyComponent', { loading: 'defer' }) }}
 
 This renders an empty ``<div>`` tag, but triggers an Ajax call to render the
 real component once the page has loaded.
 
-.. note::
+Loading "lazy"
+~~~~~~~~~~~~~~
 
-    Behind the scenes, your component *is* created & mounted during the initial
-    page load, but it isn't rendered. So keep your heavy work to methods in
-    your component (e.g. ``getProducts()``) that are only called when rendering.
+.. versionadded:: 2.17.0
+
+    The ability to load a component "lazily" was added in Live Components 2.17.
+
+The ``lazy`` option is similar to ``defer``, but it defers the loading of
+the component until it's in the viewport. This is useful for components that
+are far down the page and are not needed until the user scrolls to them.
+
+To use this, set a ``loading="lazy"`` attribute to your component:
+
+.. code-block:: html+twig
+
+    {# With the HTML syntax #}
+    <twig:Acme foo="bar" loading="lazy" />
+
+.. code-block:: twig
+
+    {# With the Twig syntax #}
+    {{ component('SomeHeavyComponent', { loading: 'lazy' }) }}
+
+This renders an empty ``<div>`` tag. The real component is only rendered when
+it appears in the viewport.
+
+Defer or Lazy ?
```

Comment:
```suggestion
Defer or Lazy?
```

I know, that's a style thing that varies by language, but without the space is the way in the Symfony docs.

Reply:
100%, just a french habit :)